src/static/major.json

```diff
@@ -487,16 +499,10 @@
       "faculties": ["DDP", "EDU"]
     },
     {
-      "code": "GLEFN",
-      "englishName": "Global Economics and Finance",
-      "chineseName": "環球經濟與金融",
-      "faculties": ["IDM", "BAF", "SSF"]
-    },
-    {
-      "code": "QFRMN",
-      "englishName": "Quantitative Finance and Risk Management Science",
-      "chineseName": "計量金融學及風險管理科學",
-      "faculties": ["IDM", "BAF", "SCF"]
```

Comment:
Same as above

Reply:
GLEFN and QFRMN were moved to a chronological order within IDM, so they are still on the list